case-lib/lib.sh

```diff
@@ -191,9 +191,8 @@ func_lib_get_tplg_path()
     return 0
 }
 
-# force ask buffer data write into file system
-sudo sync -f
-# catch kern.log last line as current case start line
-if [ ! "$DMESG_LOG_START_LINE" ]; then
-    DMESG_LOG_START_LINE=$(wc -l /var/log/kern.log|awk '{print $1;}')
+if [ ! "$CASE_KERNEL_START_TIME" ]; then
```

Comment:
I'm not a fan of the empty expression `[  ]`, do you mind the more explicit `[ -n "$CASE_KERNEL_START_TIME"  ]`?  Consistent with hijack.sh above